### ISSUE TITLE
High accuracy mode is on, but my position does not change

### ISSUE THREAD

**Issue by Hellis81** [home-assistant/android#1936]
**Home Assistant Android version:**
Beta-1626-edc4142-full

**Android version:**
11

**Phone model:**
Motorola G9 play

**Home Assistant version:**
2021.08.08

**Last working Home Assistant release (if known):**
I think this has been an issue before but never got the chance to record it

**Description of problem:**
High accuracy mode is activated using Bluetooth and I know the notification thing in Android showed a correct GPS position and was updating. 
When I got to work I looked at the lovelace and notice it still said home but the high accuracy is reported as on.
![image](https://user-images.githubusercontent.com/61010215/142148051-922abd66-ac01-4a4d-b403-a890d0c753dc.png)

To resolve this I tried to send a notification to get a request_location_update and the position was updated within a few seconds.

**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**
I downloaded the logs when I had taken the few steps between the car and entrence but that was already too late. I didn't know the logs was just one minute.
```

```

**Screenshot of problem:**

Here is the drive to work on the right compared to normal on the left.
![image](https://user-images.githubusercontent.com/61010215/142148863-e65cead3-2e41-4c79-abeb-fa2bee07b2ea.png)

**Additional information:**

**Comment by dshokouhi** [comment]
Without the logs this will be very hard to determine what went wrong

**Comment by Hellis81** [comment]
I will try and be faster to capture the logs next time it happens.
Thanks.

**Comment by dshokouhi** [comment]
So the logs are not just 1 minute long. They are whatever logs are attached to our app given that it was not killed by the system and running in the background. Sometimes they can be 1 minute and sometimes they can be 10 minutes or longer. It depends. With that said though if your position is stuck then it shouldnt take took long to get the logs to show why its stuck. Maybe some error is coming up.

**Comment by Hellis81** [comment]
I noticed the issue when I was in the car, then I resolved it and placed the phone in my pocked and walked the 30 m to my workplace and picked up the phone to get the logs and I could only get one minute of logs.
I used the share logs button to send a txt file to my mail. It only contained one minute. 
I didn't look at the logs that was in the phone, I thought (and still believe) the share logs send all the logs.

Perhaps something happened during the short walk, but the phone was not restarted and the app was still open and didn't have to reload or anything when I unlocked the phone.

We can close this for now and I will get back again if it happens again and I manage to get the logs of it.
Thank you for the help.
